partials/quarkus-extensions.adoc

```diff
@@ -0,0 +1,39 @@
+.Quarkus Extensions
```

Comment:
content of this file is basically the same as `partials/kogito-extensions.adoc`. Do we need both? Didn't we forget to update content one of them during some copy and paste? It is at least we describe the same mvn artifacts but once we say kogito extensions and once quarkus extensions

Reply:
Hmm, I don't see any references to this file or `kogito-extensions`. I'll leave these here just in case, but I'm okay leaving it as is for now.